### PR TITLE
data:daniel-jp

### PR DESCRIPTION
**Pull request by daniel-jp**
Daniel Joaquim Paulino

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3842"></a>

### CODE REVIEW

File: data/daniel-jp.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+       {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/@danieljoaquimpaulino5795/playlists",
+      "icon": "FaYoutubeSquare"
+    },
+      {
+      "name": "My virtual Badge",
+      "url": "https://daniel-jp.github.io/Badge-virtuel-mobile/",
+      "icon": "FaIdBadge"
+    },
```
Comment (Tejender1521) [inline data/daniel-jp.json:46]:
```suggestion
    }
```

**Review by Tejender1521 — CHANGES_REQUESTED** [review]

**Comment by daniel-jp** [comment]
my profile created

File: data/daniel-jp.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "Daniel J. Paulino",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software and Web Dev | Data",
+  "avatar": "https://avatars.githubusercontent.com/u/37070026?v=4",
```
Comment (Tejender1521) [inline data/daniel-jp.json:6]:
```suggestion
  "avatar": "https://github.com/daniel-jp.png",
```

**Review by Tejender1521 — CHANGES_REQUESTED** [review]

**Comment by daniel-jp** [comment]
how can I change it!?

**Pull request closed without merging**